includes/orphaned_objects.inc

```diff
@@ -54,11 +54,23 @@ function islandora_manage_orphaned_objects_form(array $form, array $form_state)
     or a variety of other reasons. Some of these orphans may exist intentionally.
     Please be cautious when deleting, as this action is irreversible.'), 'warning');
     $orphaned_objects = islandora_get_orphaned_objects();
+    $query_method = variable_get('islandora_orphaned_objects_backend', 'Solr');
+    module_load_include('inc', 'islandora', 'includes/utilities');
     $rows = array();
     foreach ($orphaned_objects as $orphaned_object) {
-      $pid = $orphaned_object['object']['value'];
+      if ($query_method == 'SPARQL') {
+        $pid = $orphaned_object['object']['value'];
+      }
+      elseif ($query_method == 'Solr') {
+        $pid = $orphaned_object['PID'];
+      }
       if (islandora_namespace_accessible($pid)) {
-        $rows[$pid] = array(l($orphaned_object['title']['value'] . " (" . $pid . ")", "islandora/object/$pid"));
+        if ($query_method == 'SPARQL') {
+          $rows[$pid] = array(l($orphaned_object['title']['value'] . " (" . $pid . ")", "islandora/object/$pid"));
+        }
+        elseif ($query_method == 'Solr') {
+          $rows[$pid] = array(l($orphaned_object['object_label'] . " (" . $pid . ")", "islandora/object/$pid"));
+        }
```

Comment:
I think you could DRY this section up by setting the object's title/label to a var in the previous "if Solr, elseif SPARQL" area, that way you can do a single `$rows[$pid] = array(l($object_title . " (" . $pid . ")", "islandora/object/$pid"));`.

Reply:
Ack, I should've got that one immediately. The lines are right above it. Silly me.